packages/web/components/Dashboard/Calendar.tsx

```diff
@@ -1,10 +1,17 @@
-import jstz from 'jstz';
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 
 const calId = 'nih59ktgafmm64ed4qk6ue8vv4@group.calendar.google.com';
```

Comment:
This should probably be in an env variable

Reply:
i moved it to `packages/web/config.ts` or should it be stored somewhere else? do we store the value into `.env` file and access it through `process.env`?